handle_latest_data.go

```diff
@@ -68,15 +68,15 @@ func handleLatestData(
 
 	debugf("* hosts identifiers: %s", identifiers)
 
-	params := Params{
-		"hostids": identifiers,
-	}
-
 	var items []Item
 	err = withSpinner(
```

Comment:
can you please place this operations (items and web scenarios) into goroutines and run it in parallel mode?